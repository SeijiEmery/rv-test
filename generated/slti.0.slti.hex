0000000    13  21  01  00  93  a1  61  ff  13  22  22  01  93  a2  f2  ff
0000020    13  23  83  3e  93  a3  03  e7  13  00  00  00  13  00  00  00
0000040    13  00  00  00  13  00  00  00  13  00  00  00  73  00  10  00
0000060
