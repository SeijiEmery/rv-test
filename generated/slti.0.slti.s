slti x2, x2, 0
slti x3, x3, -10
slti x4, x4, 18
slti x5, x5, -1
slti x6, x6, 1000
slti x7, x7, -400
nop
nop
nop
nop
nop
ebreak
