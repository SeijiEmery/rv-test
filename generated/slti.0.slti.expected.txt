tests/slti.test.s:1
R2 = 0
R3 = 0
R4 = 1
R5 = 1
R6 = 1
R7 = 0
